cli/test/test_api_wrapper.py

```
@@ -0,0 +1,184 @@
+    "reponse_class, expected, should_raise",
+    [(MockParseSuccessResponse, 1, False), (MockFailureResponse, None, True)],
+)
+def test_parse_project(api_wrapper, monkeypatch, reponse_class, expected, should_raise):
+
+    def mock_post(*args, **kwargs):
+        return reponse_class()
+
+    monkeypatch.setattr(requests, "post", mock_post)
+
+    if should_raise:
+        with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)